.ci/pytorch/common_utils.sh

```diff
@@ -142,8 +142,8 @@ function install_triton() {
   index_url=https://download.pytorch.org/whl/nightly/cpu
   if [[ "${TEST_CONFIG}" == *rocm* ]]; then
     echo "skipping triton due to rocm"
-  elif pip install "pytorch-triton==2.0.0+${short_hash}" --index-url "${index_url}"; then
-     echo "Using prebuilt version ${short_hash}"
+  # elif pip install "pytorch-triton==2.0.0+${short_hash}" --index-url "${index_url}"; then
+  #    echo "Using prebuilt version ${short_hash}"
```

Comment:
You don't need to comment those one out, it'll try to install from pip using hash and if that fails should fall back to building from source